jnigen/test/test_util/test_util.dart

```diff
@@ -88,9 +88,10 @@ void comparePaths(String path1, String path2) {
   if (diffProc.exitCode != 0) {
     final originalDiff = diffProc.stdout;
     log.warning(
-        "Paths $path1 and $path2 differ, comparing by ignoring space change");
-    final fallbackDiffProc = Process.runSync(
-        "git", [...diffCommand, '--ignore-space-change', path1, path2]);
+        "Paths $path1 and $path2 differ, Running dart format on $path1.");
+    Process.runSync('dart', ['format', path1]);
+    final fallbackDiffProc =
+        Process.runSync("git", [...diffCommand, path1, path2]);
```

Comment:
If we run the formatter, shouldn't we just be able to a literal string compare instead?

Unless you want to actually show the diff after formatting, but you're showing the originalDiff two lines below.

Reply:
If we format both files, we don't have to git diff, we can just load the two files as strings and check equality.

Reply:
There can be multiple files in a path.

Reply:
Ah, check.